pyanaconda/ui/gui/__init__.py

```diff
@@ -520,6 +521,7 @@ def __init__(self, storage, payload,
 
         self.data = None
 
+        GLib.set_prgname("liveinst")    # matches liveinst.desktop filename
```

Comment:
I don't think we should hardcode it just like this - Anaconda also runs on boot iso & can be run directly on an installed system.

I think this could be gated by `provides_liveuser()`, like we do with some other things:
https://github.com/rhinstaller/anaconda/blob/e0a528fd3f3ee4a6215e9d10fe155dfede5683c0/pyanaconda/core/configuration/system.py#L176

Reply:
What icon should those other use-cases get? Is there an `anaconda.desktop` file?

Reply:
There used to be one, but not anymore.

Reply:
```suggestion
        if provides_liveuser:
            GLib.set_prgname("liveinst")    # matches liveinst.desktop filename
```